### DISCUSSION TITLE
Is it possible to use web worker here?

### DISCUSSION THREAD

**Discussion by hojovi** [TanStack/table#3642] [Ideas]
I have a table with ten thousands of rows, when I tried to use "useTable" hook appending with other plugins like "useExpand", "useSort"... Browser seems to take too much time on constructing all the rows and the page is blocked during this time (the time spent under "useTable" call).

Since the web worker can run parallelly with js main process, is it possible to put related logic in web worker, then browser page can still handle other tasks?

**Comment by barandurakk** [comment]
Hi, I'm looking for the same thing. Did you find any solution or implemantation for this?

**Comment by MarArMar** [comment]
Looking for the same thing... Anyone ?
